tethysext/atcore/templates/atcore/resource_workflows/plot_workflow_results_view.html

```diff
@@ -0,0 +1,41 @@
+{% extends "atcore/resource_workflows/workflow_results_view.html" %}
+{% load static tethys_gizmos %}
+
+{% block content_dependent_styles %}
+  {{ block.super }}
+{% endblock %}
+
+{% block import_gizmos %}
+  {{ block.super }}
+  {% import_gizmo_dependency datatable_view %}
+{% endblock %}
+
+{% block app_content %}
+  <h1>{{ page_title }}</h1>
+  <!-- Tab panes -->
+  {% if plot_view_input %}
+    {% gizmo plot_view_input %}
+  {% else %}
+    <p>No data provided. Use add_plot or add_pandas_dataframe to create a plot.</p>
+  {% endif %}
+
+  {{ block.super }}
+{% endblock %}
+
+
+{% block scripts %}
+  {{ block.super }}
+  <script>var TAB_SELECTOR = '#dataset_tabs .nav_tabs a';</script>
+  <script src="https://cdn.datatables.net/buttons/1.6.1/js/dataTables.buttons.min.js" type="text/javascript"></script>
+  <script src="https://cdn.datatables.net/buttons/1.6.1/js/buttons.flash.min.js" type="text/javascript"></script>
+  <script src="https://cdn.datatables.net/buttons/1.6.1/js/buttons.html5.min.js" type="text/javascript"></script>
+  <script src="https://cdn.datatables.net/buttons/1.6.1/js/buttons.print.min.js" type="text/javascript"></script>
```

Comment:
I'm guessing you don't need datatables in the plot view. Please remove script includes for this.